install/common/synopsys-operator.yaml

```diff
@@ -82,6 +82,53 @@ items:
   metadata:
     name: synopsys-operator
     namespace: ${NAMESPACE}
+- kind: ClusterRole
```

Comment:
We probably want to allow the following to this role too:
- create / delete jobs